File: Java/Function2.java
import java.util.*;

class Function2
{
    public static int Addition(int Value1, int Value2)
    {
        int Result = 0;
        Result = Value1 + Value2;
        return Result;
    }
    
    public static void main(String[] args) 
    {
        int No1 = 0, No2 = 0, Ans = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter first number : ");
        No1 = sobj.nextInt();

        System.out.println("Enter first number : ");
        No2 = sobj.nextInt();

        Ans = Addition(No1, No2);

        System.out.println("Addition is : "+Ans);
        
    }
}

File: C/Function2.c
#include<stdio.h>

// Function Defination
int Addition(int Value1, int Value2)
{
    int Result = 0;                     //  Local variable

    Result = Value1 + Value2;

    return Result;
}

int main()                              // Entry point function
{
    int No1 = 0, No2 = 0, Ans = 0;      // Local variables

    printf("Enter first number : \n");
    scanf("%d",&No1);

    printf("Enter second number : \n");
    scanf("%d",&No2);
    
    Ans = Addition(No1, No2);           // Function call

    printf("Addition is : %d\n",Ans);
    
    return 0;
}